src/app/portlets/dot-edit-page/layout/components/dot-layout-properties/dot-layout-properties.component.ts

```diff
@@ -1,11 +1,12 @@
 import { DotMessageService } from '../../../../../api/services/dot-messages-service';
 import { FormGroup } from '@angular/forms';
-import { Component, Input, OnInit } from '@angular/core';
+import { Component, Input, OnInit, ViewEncapsulation } from '@angular/core';
 
 @Component({
     selector: 'dot-layout-properties',
     templateUrl: './dot-layout-properties.component.html',
-    styleUrls: ['./dot-layout-properties.component.scss']
+    styleUrls: ['./dot-layout-properties.component.scss'],
+    encapsulation: ViewEncapsulation.None
```

Comment:
You sure you need this? shouldn't we take advantage of the encapsulation of the component to style it?

Reply:
is needed, since we are doing  `appendTo="body"` to `p-overlayPanel`